zebra-chain/src/work/equihash.rs

```diff
@@ -29,16 +29,26 @@ pub struct SolverCancelled;
 /// The size of an Equihash solution in bytes (always 1344).
 pub(crate) const SOLUTION_SIZE: usize = 1344;
 
+/// The size of an Equihash solution in bytes on Regtest (always 36).
+pub(crate) const REGTEST_SOLUTION_SIZE: usize = 36;
+
 /// Equihash Solution in compressed format.
 ///
 /// A wrapper around [u8; 1344] because Rust doesn't implement common
 /// traits like `Debug`, `Clone`, etc for collections like array
 /// beyond lengths 0 to 32.
```

Comment:
###### Optional Suggestion


```suggestion
/// A wrapper around `[u8; n]` where `n` is the solution size because
/// Rust doesn't implement common traits like `Debug`, `Clone`, etc.
/// for collections like arrays beyond lengths 0 to 32.
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/8421/commits/0f91dd03df5bc3c64977c18dffbf6ecfa4bcba96.